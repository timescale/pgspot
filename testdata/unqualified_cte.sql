WITH cte1 AS (
    VALUES (1)
), cte2 AS (
    WITH foo AS (
        SELECT * FROM cte1
    )
    SELECT * FROM foo
), cte3 AS (
    SELECT * FROM foo -- This will warn, foo is not a previously defined CTE
)
SELECT * FROM cte1 AS c1
    CROSS JOIN cte2
    CROSS JOIN cte3;
