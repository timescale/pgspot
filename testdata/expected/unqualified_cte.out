PS017: Unqualified object reference: foo at line 1

 Errors: 0 Warnings: 1 Unknown: 0 

